src/applications/mhv/secure-messaging/util/actionTypes.js

```diff
@@ -28,6 +28,7 @@ export const Actions = {
     CREATE: 'SM_FOLDER_CREATE',
     DELETE: 'SM_FOLDER_DELETE',
     RENAME: 'SM_FOLDER_RENAME',
+    SET_FOLDER_ID: 'SM_FOLDER_SET_ACTIVE_FOLDER_ID',

```

Comment:
Maybe change this to 'GET_'.. it seems like you have to GET the Folder_ID in order to set it.

Reply:
Hey Vic thanks I appreciate the feedback! however the function(setActiveFolderId) that calls this isn't getting the folder Id, rather the folder id is being passed through as a parameter and is being set in state after that, so throughout the flow of this redux call there is no getting of the folder id, instead the folder Id has to have already been got prior to calling this action. You can see an example in the code in FolderThreadListView.jsx line 119.